### PR TITLE
fix(unittest): Fix regression to sticky-eviction unittest

### PR DESCRIPTION
**Pull request by chakaz**
@theyueli please make sure that with your new eviction changes, it makes sense for a recently inserted key to be removed.

### CODE REVIEW

**Review by romange — APPROVED** [review]

**Comment by chakaz** [comment]
Thanks for the approval. I'll merge this now so that we can enable CI tests again, but @theyueli please make sure that we're good in terms of logic

**Comment by chakaz** [comment]
I'll defer to you, feel free to increase coverage as you see fit :)

File: src/server/dragonfly_test.cc [hunk 1]
```diff
@@ -428,20 +428,25 @@ TEST_F(DflyEngineTest, StickyEviction) {
     ASSERT_EQ("OK", Run({"set", key, tmp_val}));
   }
 
-  for (ssize_t i = 0; i < 5000; ++i) {
+  bool done = false;
+  for (ssize_t i = 0; !done && i < 5000; ++i) {
     string key = StrCat("key", i);
-    auto set_resp = Run({"set", key, tmp_val});
-    auto stick_resp = Run({"stick", key});
+    while (true) {
```
Comment (adiholden) [inline src/server/dragonfly_test.cc:434]: instead you can run in one command "set key val stick"

Comment (theyueli) [reply src/server/dragonfly_test.cc:434]: i didn't know we could do this! this is the best solution!

Comment (chakaz) [reply src/server/dragonfly_test.cc:434]: Me neither, and best indeed! :)

**Pull request merged**